### PR TITLE
Instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR contains:
1 - instrument refurb cal csv - PHSEND
7 - Irminger 8 cruise-specific cal csvs - 2 METBKA, 2 ADCPSL, 1 ADCPSN; plus 2 NUTNRB predeploy cal csvs

### CODE REVIEW

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/ADCPSL/CGINS-ADCPSL-18446__20210803.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18446,CC_scale_factor1,0.45,constant; GI03FLMA-00008
+18446,CC_scale_factor2,0.45,constant
+18446,CC_scale_factor3,0.45,constant
+18446,CC_scale_factor4,0.45,constant
+18446,CC_bin_size,1000.0,in cm
+18446,CC_dist_first_bin,1647.0,in cm
+18446,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSL/CGINS-ADCPSL-18446__20210803.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-20503__20210803.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20503,CC_scale_factor1,0.45,constant; GI03FLMB-00008
+20503,CC_scale_factor2,0.45,constant
+20503,CC_scale_factor3,0.45,constant
+20503,CC_scale_factor4,0.45,constant
+20503,CC_bin_size,1000.0,in cm
+20503,CC_dist_first_bin,1647.0,in cm
+20503,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSL/CGINS-ADCPSL-20503__20210803.csv:8]: Looks good.

File: calibration/ADCPSN/CGINS-ADCPSN-21654__20210803.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21654,CC_scale_factor1,0.45,constant; GI01SUMO-00008
+21654,CC_scale_factor2,0.45,constant
+21654,CC_scale_factor3,0.45,constant
+21654,CC_scale_factor4,0.45,constant
+21654,CC_bin_size,1000.0,in cm
+21654,CC_dist_first_bin,1647.0,in cm
+21654,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSN/CGINS-ADCPSN-21654__20210803.csv:8]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00034__20210803.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR034,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00008
+LGR034,CC_height_of_air_humidity_measurement_m,5.255,
+LGR034,CC_height_of_air_temperature_measurement_m,5.255,
+LGR034,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR034,CC_jcool,1,Constant
+LGR034,CC_jwarm,1,Constant
+LGR034,CC_zinvpbl,600,Constant
+LGR034,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00034__20210803.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00036__20210803.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR036,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00008
+LGR036,CC_height_of_air_humidity_measurement_m,5.255,
+LGR036,CC_height_of_air_temperature_measurement_m,5.255,
+LGR036,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR036,CC_jcool,1,Constant
+LGR036,CC_jwarm,1,Constant
+LGR036,CC_zinvpbl,600,Constant
+LGR036,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00036__20210803.csv:9]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0096__20210624.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0096,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0096_Validation_Files_2021-06-24.zip > SAMI_P0096_240621_tris_ValCert.pdf
+P0096,CC_ea578,94.1,
+P0096,CC_eb434,2284.0,
+P0096,CC_eb578,38676.0,
+P0096,CC_ind_off,0,
+P0096,CC_ind_slp,1,
+P0096,CC_psal,35,Constant
```
Comment (snwhite) [inline calibration/PHSEND/CGINS-PHSEND-P0096__20210624.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20210702.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
...3.29, 274.1, 274.92, 275.73, 276.54, 277.36, 278.17, 278.98, 279.8, 280.61, 281.42, 282.24, 283.05, 283.87, 284.68, 285.49, 286.31, 287.12, 287.93, 288.75, 289.56, 290.37, 291.19, 292, 292.81, 293.63, 294.44, 295.25, 296.07, 296.88, 297.69, 298.51, 299.32, 300.13, 300.95, 301.76, 302.57, 303.38, 304.2, 305.01, 305.82, 306.64, 307.45, 308.26, 309.07, 309.89, 310.7, 311.51, 312.32, 313.13, 313.95, 314.76, 315.57, 316.38, 317.19, 318, 318.82, 319.63, 320.44, 321.25, 322.06, 322.87, 323.68, 324.49, 325.3, 326.11, 326.92, 327.73, 328.55, 329.36, 330.17, 330.98, 331.78, 332.59, 333.4, 334.21, 335.02, 335.83, 336.64, 337.45, 338.26, 339.07, 339.88, 340.68, 341.49, 342.3, 343.11, 343.92, 344.72, 345.53, 346.34, 347.14, 347.95, 348.76, 349.57, 350.37, 351.18, 351.98, 352.79, 353.6, 354.4, 355.21, 356.01, 356.82, 357.62, 358.43, 359.23, 360.04, 360.84, 361.64, 362.45, 363.25, 364.05, 364.86, 365.66, 366.46, 367.27, 368.07, 368.87, 369.67, 370.47, 371.28, 372.08, 372.88, 373.68, 374.48, 375.28, 376.08, 376.88, 377.68, 378.48, 379.28, 380.08, 380.88, 381.68, 382.47, 383.27, 384.07, 384.87, 385.67, 386.46, 387.26, 388.06, 388.85, 389.65, 390.45, 391.24, 392.04, 392.83, 393.63, 394.42, 395.22]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20210702.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20210702.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
....36, 273.18, 273.99, 274.8, 275.61, 276.42, 277.24, 278.05, 278.86, 279.67, 280.48, 281.3, 282.11, 282.92, 283.73, 284.54, 285.36, 286.17, 286.98, 287.79, 288.6, 289.42, 290.23, 291.04, 291.85, 292.66, 293.48, 294.29, 295.1, 295.91, 296.72, 297.53, 298.34, 299.16, 299.97, 300.78, 301.59, 302.4, 303.21, 304.02, 304.83, 305.65, 306.46, 307.27, 308.08, 308.89, 309.7, 310.51, 311.32, 312.13, 312.94, 313.75, 314.56, 315.37, 316.18, 316.99, 317.8, 318.61, 319.42, 320.23, 321.04, 321.85, 322.66, 323.47, 324.27, 325.08, 325.89, 326.7, 327.51, 328.32, 329.13, 329.93, 330.74, 331.55, 332.36, 333.16, 333.97, 334.78, 335.59, 336.39, 337.2, 338.01, 338.81, 339.62, 340.43, 341.23, 342.04, 342.84, 343.65, 344.46, 345.26, 346.07, 346.87, 347.68, 348.48, 349.29, 350.09, 350.9, 351.7, 352.5, 353.31, 354.11, 354.91, 355.72, 356.52, 357.32, 358.13, 358.93, 359.73, 360.53, 361.33, 362.14, 362.94, 363.74, 364.54, 365.34, 366.14, 366.94, 367.74, 368.54, 369.34, 370.14, 370.94, 371.74, 372.54, 373.34, 374.14, 374.94, 375.73, 376.53, 377.33, 378.13, 378.92, 379.72, 380.52, 381.31, 382.11, 382.91, 383.7, 384.5, 385.29, 386.09, 386.88, 387.68, 388.47, 389.26, 390.06, 390.85, 391.65, 392.44, 393.23, 394.02]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20210702.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20210702.csv (hunk 8 above)
Comment (reedan88) [reply calibration/NUTNRB/CGINS-NUTNRB-01092__20210702.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20210702.csv (hunk 7 above)
Comment (reedan88) [reply calibration/NUTNRB/CGINS-NUTNRB-01093__20210702.csv:8]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0096__20210624.csv (hunk 6 above)
Comment (reedan88) [reply calibration/PHSEND/CGINS-PHSEND-P0096__20210624.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
Everything looks good.

**Pull request merged**